src/shared/components/common/markdown-textarea.tsx

```diff
@@ -346,60 +348,74 @@ export class MarkdownTextArea extends Component<
   }
 
   handleImageUpload(i: MarkdownTextArea, event: any) {
-    let file: any;
-    if (event.target) {
-      event.preventDefault();
-      file = event.target.files[0];
-    } else {
-      file = event;
+    i.setState({ imageLoading: true });
+    // TODO: not sure if this is needed
+    if (!event.target) {
+      console.log("event");
+      i.uploadImage(i, event)
+      i.setState({ imageLoading: false });
+      return
+    } 
+    if (event.target.files.length > 20) {
```

Comment:
Don't hardcode 20, create a constant in `utils.ts` and import it.

Reply:
Of course.